SignalServiceKit/src/Storage/OWSMediaGalleryFinder.m

```diff
@@ -147,17 +147,20 @@ + (YapDatabaseAutoView *)mediaGalleryDatabaseExtension
             return nil;
         }
         TSMessage *message = (TSMessage *)object;
-        
-        OWSAssertDebug(message.attachmentIds.count <= 1);
-        NSString *attachmentId = message.attachmentIds.firstObject;
-        if (attachmentId.length == 0) {
-            return nil;
+
+        BOOL allAttachmentsAreMedia = message.attachmentIds.count > 0;
```

Comment:
I did the simplest possible changes to get the media views working; there's more follow-up work.  Right now:

* We only treat messages as media messages if _all_ of their attachments are media (should be true).
* I think the media views will only use the first attachment from multi-image messages, but I haven't tested it yet.